File: consultans_sales/src/main/java/consultan/vanke/com/viewmodel/IzuiyouViewModel.java
package consultan.vanke.com.viewmodel;

import android.app.Application;
import consultan.vanke.com.bean.Izuiyou;
import io.reactivex.Maybe;

import java.util.List;

public class IzuiyouViewModel extends BaseViewModel {

    public IzuiyouViewModel(Application application) {
        super(application);
    }

    public Maybe<List<Izuiyou>> getListData() {
        return netApi.getIzuiyou();

    }

}

File: consultans_sales/src/main/java/consultan/vanke/com/viewmodel/kt/IzuiyouViewModel.kt
package consultan.vanke.com.viewmodel.kt

import android.app.Application
import consultan.vanke.com.bean.kt.Izuiyou
import consultan.vanke.com.viewmodel.BaseViewModel
import io.reactivex.Maybe

class IzuiyouViewModel(application: Application?) : BaseViewModel(application!!) {
    val listData: Maybe<List<Izuiyou>>
        get() = netApi.izuiyouKt
}